src/Modal.tsx

```diff
@@ -38,7 +38,7 @@ export const ModalProvider: React.FC<ModalProviderProps> = ({
   const showModal = useCallback(
     (content: React.ReactNode, options?: Partial<ModalProps>) => {
       setContent(content);
-      if (options) optionsRef.current = options;
+      optionsRef.current = options;
```

Comment:
Is it okay to assign falsy values?

Reply:
it could be undefined in that case. which is fine.